lib/workers/repository/init/apis.ts

```diff
@@ -24,6 +51,8 @@ export async function initApis(
 ): Promise<WorkerPlatformConfig> {
   let config: WorkerPlatformConfig = { ...input } as never;
   config = await getPlatformConfig(config as never);
+  await validateOptimizeForDisabled(config);
+  await validateIncludeForks(config);
```

Comment:
Shouldn't we merge those function, otherwise we do two requests for the same file. 🤔

Reply:
The cache should catch it so not be much overhead. `optimizeForDisabled` should not be likely to be in common use either.